### PR TITLE
Te PIGS demo

### PR DESCRIPTION
**Pull request by venkatkapil24**

### CODE REVIEW

File: demos/te-pigs/README.md
Comment (ceriottm) [inline demos/te-pigs/README.md:1]: README is empty

**Comment by ceriottm** [comment]
Hey. This looks harmless enough but please do add a README and include in the README your contacts so if people have problems they know who to pester. We should also add those for the other examples, I'll do a flash-PR for that

**Comment by venkatkapil24** [comment]
yes, this is still in preparation.

**Comment by venkatkapil24** [comment]
@ceriottm ping.

**Review by ceriottm — APPROVED** [review]

**Pull request merged**